packages/run-protocol/src/vpool-xyk-amm/types.js

```diff
@@ -83,8 +91,9 @@
  */
 /**
  * @typedef {object} XYKAMMPublicFacet
- * @property {(issuer: ERef<Issuer>, keyword: Keyword) => Promise<Issuer>} addPool
+ * @property {() => Promise<Invitation>} addPoolInvitation
```

Comment:
the `Invitation` can be typed with the offer args and result. I don't know it offhand but I'd be happy to help.

Reply:
nevermind, I tried a bit and it's more effort than I thought.